lib/compiler.js

```diff
@@ -1,62 +1,102 @@
 var path = require("path");
```

Comment:
The PR lists several files, including this one, as having “conflicts that must be resolved”.

Reply:
Hi @JohnAlbin,

I'm happy to make the required changes, but it would be good to know if this PR is going to be used at all in any way, since the original author seems afk after I made the requested changes :)

Let me know! Cheers